### ISSUE TITLE
Does it work?

### ISSUE THREAD

**Issue by nahuelange** [Y300-0100/android_device_samsung_G355HN#2]
Is this cm mod for g355hn is working?

**Comment by Y300-0100** [comment]
Yes but not working.

**Comment by nahuelange** [comment]
What is missing?

**Comment by Y300-0100** [comment]
Proprietary blobs and rule to make kernel in board.mk

**Comment by noahp78** [comment]
Since it has been a while I'm going to ask again: does it work now?

**Comment by Y300-0100** [comment]
Last device tree for kanas3gnfcxx and vendor is ok but because of kernel I got bootlop. So if u got any idea please suggest.

**Comment by nahuelange** [comment]
Don't samsung provide kernel config file ?

**Comment by Y300-0100** [comment]
Yes but even with default config can't build proper bootable kernel(for stock rom). They probably added something else and didn't released it.

**Comment by nahuelange** [comment]
Do you use the kernel provided by samsung or another one ?

**Comment by Y300-0100** [comment]
Yes, kernel source by samsung. Its on my github.

**Comment by nahuelange** [comment]
Did you try to contact samsung and said their kernel does'nt work ? may be they can release more?

**Comment by nahuelange** [comment]
And, can't you get the kernel already build in stock rom ?
